packages/toolpad-app/src/server/handleDataRequest.ts

```diff
@@ -29,6 +29,18 @@ export default async (
   const apiNodeId = req.query.queryId as NodeId;
   const dom = await loadVersionedDom(appId, version);
   const api = appDom.getNode(dom, apiNodeId, 'api');
+  if (api.attributes.transformEnabled.value) {
```

Comment:
Won't this fail for existing apis in toolpad? Their `transformEnabled` property and `transform` property will be `undefined`

Reply:
Yes, correct - should use optional chaining for the `transform` and `transformEnabled` accesses everywhere